frontend/src/components/maps/leaflet/LayerPopup/hooks/useLayerQuery.tsx

```diff
@@ -84,6 +84,9 @@ export const useLayerQuery = (url: string, geometryName: string = 'SHAPE'): IUse
     async (pid: string): Promise<FeatureCollection> => {
       //Do not make a request if we our currently cached response matches the requested pid.
       const formattedPid = pid.replace(/-/g, '');
+      if (isNaN(+formattedPid)) {
```

Comment:
never run a pid search with a non-numeric pid.

Reply:
Why the `+`?

Reply:
+ is js syntax for integer parsing. Here is some doc:
The + and - operators also have unary versions, where they operate only on one variable. When used in this fashion, + returns the number representation of the object, while - returns its negative counterpart.

https://en.wikibooks.org/wiki/JavaScript/Operators#Arithmetic_operators